packages/@expo/config-plugins/src/utils/Updates.ts

```diff
@@ -37,8 +37,13 @@ export function getUpdateUrl(
   return `https://exp.host/@${user}/${config.slug}`;
 }
 
-export function getAppVersion(config: Pick<ExpoConfig, 'version'>): string {
-  return config.version ?? '1.0.0';
+export function getAppVersion(
```

Comment:
we might want to rename this method to `getAppVersionRuntimeVersion` (so other places don't accidentally think this returns the app version). Also rename `getNativeVersion` to `getNativeVersionRuntimeVersion`.